packages/api/src/did/did-doc.ts

```diff
@@ -0,0 +1,24 @@
+import { DidDocument, didDocument } from '@atproto/common-web'
+
+export function isValidDidDoc(doc: unknown): doc is DidDocument {
```

Comment:
not required, but I think we could take this one step further & move this to common-web as well